cypress/e2e/data-import/settings/check-vendor-mapping-ui.cy.js

```diff
@@ -0,0 +1,98 @@
+import {
+  FOLIO_RECORD_TYPE,
+  ORDER_STATUSES,
+  VENDOR_NAMES,
+  ACQUISITION_METHOD_NAMES,
+  ORDER_FORMAT_NAMES_IN_PROFILE,
+} from '../../../support/constants';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import FieldMappingProfiles from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import Users from '../../../support/fragments/users/users';
+import { Permissions } from '../../../support/dictionary';
+import SettingsMappingProfiles from '../../../support/fragments/settings/dataImport/settingsMappingProfiles';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import FieldMappingProfileEdit from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileEdit';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    const testData = {};
+    const mappingProfile = {
+      name: `C367955testMappingProfile.${getRandomPostfix()}`,
+      typeValue: FOLIO_RECORD_TYPE.ORDER,
+      orderStatus: ORDER_STATUSES.PENDING,
+      vendor: VENDOR_NAMES.GOBI,
+      title: '245$a',
+      acquisitionMethod: ACQUISITION_METHOD_NAMES.PURCHASE_AT_VENDOR_SYSTEM,
+      orderFormat: ORDER_FORMAT_NAMES_IN_PROFILE.PE_MIX,
+      receivingWorkflow: 'Synchronized',
+      currency: 'USD',
+      materialSupplier: 'Amazon.com',
+      accessProvider: 'EBSCO',
+      incomingRecordType: 'MARC_BIBLIOGRAPHIC',
+      recordType: 'ORDER',
+      description: '',
+    };
+    const sectionValues = [
+      { sectionName: 'Vendor', value: 'GOBI Library Solutions', isRequired: true },
+      { sectionName: 'Material supplier', value: 'Amazon.com', isRequired: false },
+      { sectionName: 'Access provider', value: 'EBSCO', isRequired: false },
+    ];
+
+    before('Create test data', () => {
+      cy.createTempUser([
+        Permissions.settingsDataImportEnabled.gui,
+        Permissions.uiOrganizationsView.gui,
+      ]).then((userProperties) => {
+        testData.user = userProperties;
+        cy.login(testData.user.username, testData.user.password, {
+          path: SettingsMenu.mappingProfilePath,
+          waiter: FieldMappingProfiles.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(testData.user.userId);
+      FieldMappingProfiles.getFieldMappingProfileInDataImport({
```

Comment:
For deleting mapping profile is function like FieldMappingProfileView.deleteViaApi(profileName);

Reply:
Fixed, thank you!